src/pixel_display.toit

```diff
@@ -743,17 +748,18 @@ class GrayScalePixelDisplay extends PixelDisplay:
     height := 0
     // Keep each color component under 2k so you can fit two on a page.
     height = round_down (2000 / width) 8
-    // We can't work well with canvases that are less than 8 pixels tall.
-    return max 8 height
+    // We can't work well with canvases that are less than 4 pixels tall.
+    return height < 8 ? 4 : height
```

Comment:
The comment could need a bit more update (although I'm probably fine with it).
It's not just that canvases with less than 4 pixels are "upgraded" to "4". Every value below 8 is changed to 4.